IMF Vanilla 0.99 AGAIN!

fwlekrjnfrw opened the pull request:
Uhhh hope this works!
Big changes are obviously no rares save for cards you can only get one of in act 2
one candle, the player only gets one in act 2
Wolf is banned, surprisingly you can draft it in act 2
And we all love the spore cards, right?

fwlekrjnfrw committed:
Add files via upload

fwlekrjnfrw committed:
Update vanilla.json

fwlekrjnfrw committed:
Delete IMF Vanilla v0.99.json

fwlekrjnfrw commented:
I meant CAN'T draft it, whoops

fwlekrjnfrw committed:
green mage attack fix

Green mage's attack now only buffs off of green mox like in act 2

107zxz commented:
Test comment for the discord bot :)

fwlekrjnfrw commented:
Yippee it works! Also for the most part this is as close as the ruleset can get to being vanilla act two and thus wont need any updates (other than maybe sorting the order like in my consistency update idea) so would you consider this IMF Vanilla 1.0? There's still some bug fixes like Spore digger not working...

107zxz commented:
Finally got a chance to check this out properly. It's great! Thank you so so much and so sorry for the delay

Mouthless-Stoat commented:
If you don't mind. All the tab character should be replace with 4 spaces. This is mainly style and to reduce the diff size.

fwlekrjnfrw commented:
> If you don't mind. All the tab character should be replace with 4 spaces. This is mainly style and to reduce the diff size.

Will get to that soon!

fwlekrjnfrw committed:
Saving space and saving face

Replaced every instance of tab with four spaces by Mouthless Stoat's request, fixed sigils/mox cost taking 3/4 lines instead of 1, and hopefully fixed stoat using the act 2 themed act 1 sprite instead of using the act 2 sprite

fwlekrjnfrw commented:
> If you don't mind. All the tab character should be replace with 4 spaces. This is mainly style and to reduce the diff size.

Should be done, threw in some other changes as well

Mouthless-Stoat commented:
Oh, I lie, it is 2 spaces lmao. You don't have to fix it I can fix it when we merge.

107zxz commented:
In that case I think a merge is in order
